### PR TITLE
ResNet-50 || ImageNet Training on Tensorflow 1.11 || GE Healthcare

### PR DESCRIPTION
**Pull request by mzhangatge**
@codyaustun 

ResNet-50 trained on **single  p3.16xlarge (8xlarge)** (using Amazon AWS Sagemaker services), the goal is to train the imagenet using public affordable resources using single instance.

Thank you!

### CODE REVIEW

File: ImageNet/train/gehc_resnet50_1_p3_16xlarge_tensorflow111.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
+{
+    "version": "v1.0",
+    "author": "GE Healthcare EdisonAITK",
+    "authorEmail": "Min.Zhang@ge.com",
+    "framework": "tensorflow 1.11 + horovod",
+    "codeURL": "https://github.com/mzhangatge/imagenet106",
+    "model": "Resnet 50 v1",
+    "hardware": "8*V100 (AWS Sagemaker p3.16xlarge)",
+    "costPerHour": 24.48,
+    "timestamp": "2018-12-30",
```
Comment (codyaustun) [inline ImageNet/train/gehc_resnet50_1_p3_16xlarge_tensorflow111.json]: Please update this to `2018-01-06`, which is the date the pull request was open.

File: ImageNet/train/gehc_resnet50_1_p3_16xlarge_tensorflow111.json [hunk 2]
```diff
@@ -0,0 +1,29 @@
+{
+    "version": "v1.0",
+    "author": "GE Healthcare EdisonAITK",
+    "authorEmail": "Min.Zhang@ge.com",
+    "framework": "tensorflow 1.11 + horovod",
+    "codeURL": "https://github.com/mzhangatge/imagenet106",
```
Comment (codyaustun) [inline ImageNet/train/gehc_resnet50_1_p3_16xlarge_tensorflow111.json]: Please specify the commit hash either in the `codeURL` or as a separate `commitHash` field

**Review by codyaustun — CHANGES_REQUESTED** [review]
Looks good overall. Just need a few small changes.

File: ImageNet/train/gehc_resnet50_1_p3_16xlarge_tensorflow111.json (hunk 2 above)
Comment (mzhangatge) [reply ImageNet/train/gehc_resnet50_1_p3_16xlarge_tensorflow111.json]: thank you, the corresponding changes have been made.

File: ImageNet/train/gehc_resnet50_1_p3_16xlarge_tensorflow111.json (hunk 1 above)
Comment (mzhangatge) [reply ImageNet/train/gehc_resnet50_1_p3_16xlarge_tensorflow111.json]: thank you, the corresponding change has been made

**Comment by mzhangatge** [comment]
@codyaustun thank you, the corresponding changes have been made.

**Review by codyaustun — APPROVED** [review]

**Pull request merged**